Formula/wildfly-as.rb

```diff
@@ -61,4 +61,28 @@ def plist; <<~EOS
     ENV["JBOSS_HOME"] = opt_libexec
     system "#{opt_libexec}/bin/standalone.sh --version | grep #{version}"
   end
+
+  test do
```

Comment:
If you're going to leave the previous test in place, the `test do` block should be merged so there's just one.